app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -156,7 +156,7 @@ def open_claims
   # fetch, memoize, and return all of the veteran's rated disabilities from EVSS
   def all_rated_disabilities
     settings = Settings.lighthouse.veteran_verification.form526
-    icn = UserAccount.where(id: user_account_id).first&.icn
+    icn = UserAccount.where(id: user_uuid).first&.icn
```

Comment:
So I think this was actually a bug, as far as I can tell there is no `user_account_id`

Reply:
@sethdarragile6 if you could take a look at this for me it would be much appreciated

Reply:
this is a `concern` file. 
the `concern` is attached to `Form526Submission`. 
`Form526Submission` `belongs_to :user_account` (line 59-ish), therefore, it has a `user_account_id`.

`user_uuid` is not the same as a UserAccount `id`, which is the table ActiveRecord is searching on.

this should probably change back to `user_account_id` or else it will never be found :sweat_smile:

Reply:
this is a `concern` file. 
the `concern` is attached to `Form526Submission`. 
`Form526Submission` `belongs_to :user_account` (line 59-ish of Form526Submission.rb), therefore, it has a `user_account_id`.

`user_uuid` is not the same as a UserAccount `id`, which is the table ActiveRecord is searching on.

this should probably change back to `user_account_id` or else the UserAccount will never be found :sweat_smile:

Reply:
Good point, none of the examples passed in the `user_account` and it wasn't mentioned as an attribute in the comments in `form526_submission.rb` so I was having trouble getting a non-nil value for `user_account_id`. I only updated the portion of the spec that was related to the changes I made in the PR, but I can update the other instances where `Form526Submission` objects are created as well if you would like. Either way, thanks for catching this!